main.tf

```diff
@@ -1,3 +1,12 @@
+data "aws_subnet" "this" {
+  for_each = toset(var.subnet_ids)
+  id       = each.value
+}
+
+locals {
```

Comment:
Wow, amazing that this is our only `locals` block in this `main`. Nothing to do here, but that's unusual.

Reply:
There was actually another one. I've moved this into the other.